README.md

```diff
@@ -1 +1,182 @@
-# Math-magician
\ No newline at end of file
+# Math-magician
+<a name="readme-top"></a>
+
+<div align="center">
+
+  # Nji Wilsan N.
+  <br/>
+
+  <h3><b>Math Magicians Project</b></h3>
+
+</div>
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+
+# 📖 MATH-MAGICIANS
+
+This is my first Micronaut React project. Some complex coding is required for this exercise. The goal is to master all of the tools and best practices. 
+
+![Preview Image](Images/preview.png)
+
```

Comment:
- [ ] It would be great if you fix the broken link to make your readme descriptive and professional 👍